crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -58,9 +58,7 @@ def upgrade_error_flash(message)
     end
 
     def upgrade_ha_repo_needed?
-      return true if Proposal.where(barclamp: "pacemaker").first
-
-      false
+      !!Proposal.where(barclamp: "pacemaker").first
```

Comment:
I don't like this coding style, it's much harder to read. The existing code is the common ruby style.